src/components/BookList/bookList.js

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect } from 'react';
+// import './bookList.css';
```

Comment:
- [ ] Following the best practices, you are not meant to leave unused code as a comment in your codebase, I'll suggest you remove this as it makes the codebase look a bit unprofessional. :smile: :wave:

Reply:
@V-Blaze  I have fixed the issues as you mentioned in the review.